webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+
+const production = process.env.NODE_ENV === 'production';
```

Comment:
this differs from listenbrainz - https://github.com/metabrainz/listenbrainz-server/blob/3850306efc3674ba1601429862280049102bedae/webpack.config.js#L6
which takes `env` as an argument to module.exports. How do these methods differ?

Reply:
As we discussed on IRC, this could be solved by exporting a function rather than an object, and using env arguments passed to the cli command: https://webpack.js.org/guides/environment-variables/#root

Reply:
I'll update it to reflect the LB setup for some consistency, and this will make it easier to do this, thanks